osu.Game.Rulesets.Osu.Tests/Editor/TestSceneObjectMerging.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
Should probably remove this, we avoid adding it for new files.

Reply:
This comment concerns `#nullable disable` and not the copyright notice. [b5e5411](https://github.com/ppy/osu/pull/19782/commits/b5e54113485d5c511bc179a67a2f0794cd78ec55) is wrong.

@OliBomby Please do not resolve review conversations as the author of the code. In the way we work, the reviewer (or another maintainer) is responsible for this.

Reply:
oh sorry, i blundered